### ISSUE TITLE
{Linked:Bug4647877}[Adaptive Card- Designer] - Talkback is announcing irrelevant information while navigating on the progress bar present on the poll card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#451]
### Target Platforms

Android

### SDK Version

1416/1.0.0.2025140801/0715

### Application Name

Teams

### Problem Description

**Test Environment:** 
Platform: Android (Samsung Galaxy S24) 
Platform version: 15 
Application: Microsoft Teams 
Teams' version: 1416/1.0.0.2025140801/0715 (Ring0) 
Calling Version: 2025.23.01.10 
Screen Reader: Talkback 

**Repro steps:** 
1. Open the Teams application. 
2. Navigate to the 'Chat' tab through swipe gesture and double tap to activate it. 
3. Navigate to any chat with Adaptive cards shared over the chat. 
4. Navigate to the 'AC Web Render Test' group chat through swipe gesture and double tap to activate it.. 
5. Navigate inside the poll adaptive card through swipe gesture and double tap to activate it. 
6. Navigate with swipe gesture.. 
7. Observe the issue. 

**Actual Result:** 
Talkback is announcing irrelevant information while navigating on the progress bar present on the poll card. Here it is announcing the link information along with image. 

**Observation:** 
Same issue observed for same adaptive card controls inside chat, channel and Channel reply chain. 
 
**Expected Result:** 
Talkback should announce the correct and relevant information while navigating on the progress bar. It should announce something like "Progress indicator/ bar, 22 %" 

**User Experience:** 
Screen reader users will get impacted if the talkback announces irrelevant information on the progress bar inside the polls adaptive cards, due to which users will not understand the proper information of the progress bar. 

**WCAG Reference:** 
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships 

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/4b8e82bc-535d-47c5-8131-a0decb7f5c0b

<img width="360" height="789" alt="Image" src="https://github.com/user-attachments/assets/3cd29be5-2de3-4dfa-95b8-83aafc0e48c0" />

### Card JSON

```JSON
NA
```
